fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -99,7 +104,10 @@ def execute(command, hide_output)
         UI.important("Although errors occurred during execution of iTMSTransporter, it returned success status.")
       end
 
-      exit_status.zero?
+      {
```

Comment:
@mnespor Instead of changing this return type, could we instead add an optional block that is passed into this `execute` method where you can parse the `@all_lines`? Changin this return type is a bit spooky 😱

Reply:
Definitely